### ISSUE TITLE
Unable to register/login from mobile view

### ISSUE THREAD

**Issue by tildearrow** [phoronix-test-suite/openbenchmarking#5]
Because the dropdown menu button does not exist anymore, it is impossible to register or login from a phone.

**Comment by Forza-tng** [comment]
Yes I saw that too. In my browser I can force enable desktop mode. Unfortunately I got the same bug as others, in that I didn't receive the activation email.
